solution/ui/e2e/cypress/e2e/subjects.spec.cy.js

```diff
@@ -608,57 +608,11 @@ describe("Find by Subjects", () => {
     });
 
     it("should have the correct labels for public and internal documents", () => {
-        cy.intercept("**/v3/content-search/?q=test**", {
-            fixture: "policy-docs.json",
-        }).as("queriedFiles");
-        cy.viewport("macbook-15");
-        cy.eregsLogin({
+        cy.checkPolicyDocs({
             username,
```

Comment:
Since the Resources column (the right column) on the Search page now requests and displays data the exact same way as the Subjects page, I've moved the assertions for those results to a utility method that can be called in both the Search and Subjects test suites.